store/store.go

```diff
@@ -173,6 +176,16 @@ type Store struct {
 	suggestedCurrency string
 }
 
+func httpStatusErrToError(httpErr httpErrorResponse, msg string) error {
+	tpl := "cannot %s: got unexpected HTTP status code %d via %s to %q"
+	if httpErr.Oops != "" {
+		tpl += " [%s]"
+		return fmt.Errorf(tpl, msg, httpErr.StatusCode, httpErr.Method, httpErr.URL, httpErr.Oops)
+	}
+
```

Comment:
You're right, thanks for spotting, this function needs to return an error also for the non-oops cases. Fixed.